src/components/header/Header.jsx

```diff
@@ -1,10 +1,7 @@
 import * as React from 'react';
-import AppBar from '@mui/material/AppBar';
-import Box from '@mui/material/Box';
-import Toolbar from '@mui/material/Toolbar';
-import Typography from '@mui/material/Typography';
-import Button from '@mui/material/Button';
-import IconButton from '@mui/material/IconButton';
+import {
+  AppBar, Box, Toolbar, Typography, Button, IconButton,
```

Comment:
if you have more than 3 imports made this like this
{
...
...
...
...
...} import from....